tests/e2e/backup_restore_suite_test.go

```diff
@@ -176,7 +176,7 @@ func runApplicationBackupAndRestore(brCase ApplicationBackupRestoreCase, expecte
 	}
 }
 
-func runBackup(brCase BackupRestoreCase, backupName string) bool {
+func runBackup(brCase BackupRestoreCase, backupName string, expectedErr error) bool {
```

Comment:
I guess technically not, since I haven't added any tests with errors in the backup stage. But it looked weird to have it all plumbed through runRestore and not runBackup.